recipes/pixelator/meta.yaml

```diff
@@ -0,0 +1,72 @@
+{% set name = "pixelator" %}
+{% set version = "0.13.1" %}
+{% set pypi_name = "pixelgen-pixelator" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ pypi_name[0] }}/{{ pypi_name }}/{{ pypi_name.replace('-', '_') }}-{{ version }}.tar.gz
+  sha256: b15d41f33c5f2dc29c9b65436c4a8c4551364c0dc0daa7cc780b10e818318753
```

Comment:
If you like, you can use GitHub instead - I'm not sure if there are pros / cons of either 🤔 

eg. https://github.com/bioconda/bioconda-recipes/blob/14498dcd177df52f5a54df5dbecf0b4d7b9bf921/recipes/multiqc/meta.yaml#L8-L9